tools/memap.py

```diff
@@ -486,13 +495,13 @@ def main():
 
     parser.add_argument('file', help='memory map file')
 
-    parser.add_argument('-t', '--toolchain', dest='toolchain', help='select a toolchain used to build the memory map file (ARM, GCC_ARM, IAR)',\
-                        required=True)
+    parser.add_argument('-t', '--toolchain', dest='toolchain', help='select a toolchain used to build the memory map file (%s)' % ", ".join(MemapParser.toolchains),\
```

Comment:
The only issue with this is for the ARM toolchain. In all of the CLI scripts, ARM==ARM_STD and uARM==ARM_MICRO. I'm not familiar with this script and what exactly it expects, perhaps @mlnx can elaborate a bit?

But if this script is to be invoked by the user, than they should be using `ARM` and `uARM`, not `ARM_STD` and `ARM_MICRO`.